rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -604,5 +572,160 @@
                 <matches pattern="(TableGenerator|TableHiLoGenerator|SequenceGenerator|SequenceIdentityGenerator|SequenceHiLoGenerator)" />
             </where>
         </rule>
+        <rule id="hibernate4-00029">
+            <when>
+                <javaclass references="org.hibernate.type.descriptor.sql.SqlTypeDescriptor">
```

Comment:
I'm not sure this will work correctly. The text refers to "extending built-in SqlTypeDescriptors" -- is this rule going to match them?